src/firefly/js/visualize/ui/ImageCenterDropDown.jsx

```diff
@@ -0,0 +1,166 @@
+    return strAry.map( (s) => parseWorldPt(s));
+}
+
+
+function  hasTablesWithCoordinates(pv) {
+    const group= getTableGroup();
+    if (!group || !pv) return false;
+    const tAry= Object.values(group.tables);
+    const coordTable= tAry.find( (t) => t);
+    return Boolean(pv && coordTable);
+}
+
+function prettyTarget(wp) {
+    if (wp.objName) {
+        if (wp.resolver) {
+            return (
+                <Fragment>
+                    <span style={{fontWeight:'bold'}}>{wp.objName}</span>
+                    <span style={{fontSize: '80%' }}>
+                        <span style={{fontStyle: 'italic'}}> by </span>
+                        {`${wp.resolver.toString().toUpperCase()}`}
+                    </span>
+                </Fragment>
+            );
+        }
+        else {
+            return ( <span style={{fontWeight:'bold'}}>{wp.objName}</span> );
+        }
+    }
+    else {
+        return ( <span style={{fontWeight:'bold'}}>{`${wp.x}, ${wp.y} ${coordToString(wp.cSys)}`}</span> );
+    }
+}
+
+
+export function ImageCenterDropDown({visRoot:vr, visible}) {
+
+    const pv= getActivePlotView(vr);
+    const plot= primePlot(pv);
+    const coordTables= hasTablesWithCoordinates(pv);
+    const hasSearchTarget= Boolean(get(plot, ['attributes',PlotAttribute.FIXED_TARGET]));
+    const plotId= plot && plot.plotId;
+    const [recentAry, setRecentAry] = useState(getRecentTargets());
+
+    const moveToTarget= (fields) => {
+        const wp=  parseWorldPt(get(fields,'UserTargetWorldPt'));
+        if (!wp) return;
+        dispatchRecenter({plotId, centerPt:wp});
+        addToRecentTargets(wp, setRecentAry);
+    };
+    //   //todo code to call a future marker option
+    // const createMarkerAndMoveToTarget= (fields) => {
+    //     const wp=  parseWorldPt(get(fields,'UserTargetWorldPt'));
+    //     if (!wp) return;
+    //     moveToTarget(fields);
+    //                              todo: the section below might need to be deferred to the plot has moved to the target first
+    //     const cc = CysConverter.make(plot);
+    //     if (cc.pointInPlot(wp)) {
+    //         const title= prettyTarget(wp);
+    //         const drawLayerId = title;
+    //         dispatchCreateMarkerLayer(drawLayerId, title, plotId, true, wp, true);
+    //     }
+    // };
+    //
+
+    const dropDown= (
+        <SingleColumnMenu>
+
+            <ToolbarButton text='Pan by table row' tip='Center selected images position of the highlighted row'
+                           enabled={coordTables}
+                           horizontal={false} key={'pan-table'}
+                           hasCheckBox={true} checkBoxOn={vr.autoScrollToHighlightedTableRow}
+                           onClick={() => dispatchChangeTableAutoScroll(!vr.autoScrollToHighlightedTableRow)}/>
+
+            <DropDownVerticalSeparator useLine={true}/>
+
+
+            <ToolbarButton text='Center on Target' tip='Center on search target'
```

Comment:
Trying to come to this with a blank slate, I found it confusing to think about what the user will understand as a/the "target" or even "search target".  It is *very* easy to imagine a user thinking of what they typed in the "Center On..." field as a "target".

What is really is, I think, is the name or coordinates most recently used in an image or catalog search, is that right?  Let's see if we can think of a way to make that clearer.

Reply:
Just let me know what the text should be and I will set it to that.

Reply:
I do not have an actionable suggestion for now.  We can address this in a future ticket if the CCB-D has a good idea.